### ISSUE TITLE
Datatype missing on v12 with Ubuntu 20.04

### ISSUE THREAD

**Issue by gamesguru** [cbbrowne/autodoc#22]
Hi, receiving some error messages and missing data types for columns on `.eps` exports.

Repeats dozens of times:

```
DBD::Pg::st fetchrow_hashref failed: no statement executing at /usr/bin/postgresql_autodoc line 689.
DBD::Pg::st execute failed: ERROR:  column "adsrc" does not exist
LINE 40:                 adsrc
                         ^ at /usr/bin/postgresql_autodoc line 687.
DBD::Pg::st fetchrow_hashref failed: no statement executing at /usr/bin/postgresql_autodoc line 689.

Use of uninitialized value in numeric comparison (<=>) at /usr/bin/postgresql_autodoc line 991.
Use of uninitialized value in numeric comparison (<=>) at /usr/bin/postgresql_autodoc line 1466.
```

**Comment by cbbrowne** [comment]
I think this is the same thing as issue #19 which was fixed in commit f4d53c180350905648efd4b3ef09e2bfeb7d5a4b

**Comment by Dmole** [comment]
@cbbrowne Please prod the Debian/Ubuntu Maintainer.
